### PR TITLE
Add YouTube support, add Dan Vega

### PR DESCRIPTION
**Pull request by scordio**
Taking inspiration from #264, I've added support for YouTube channels. However, I don't have experience with the YouTube API and I'm not sure what the correct way would be to validate handles.

As a showcase, I've added Dan Vega who was generally missing:

Screenshot 2023-10-10 at 00 25 32

Originally discussed at https://x.com/stefanocodes/status/1707277470800453701

### CODE REVIEW

File: generator/src/main/java/org/javabubble/generator/model/YouTubeHandle.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.javabubble.generator.model;
+
+import java.util.regex.Pattern;
+
+public final class YouTubeHandle extends Handle {
+
+	private static final Pattern HANDLE_PATTERN = Pattern.compile("[A-Za-z0-9_.-]{3,30}");
```
Comment (scordio) [inline generator/src/main/java/org/javabubble/generator/model/YouTubeHandle.java:7]: Pattern taken from [Handle naming guidelines](https://support.google.com/youtube/answer/11585688).

**Comment by marchof** [comment]
@scordio Thanks for this nice contribution!

**Comment by scordio** [comment]
@marchof would you consider [joining Hacktoberfest](https://hacktoberfest.com/participation/#maintainers) with this repo?

**Comment by marchof** [comment]
@scordio Unfortunatelly I can't spend too much time in this side project. Which means:

1) I can't support Hacktoberfest activities personally
2) I want to keep the scope and functionality of this repository small (--> minimize maintainance effort)

Do you have a specific contribution in mind? The only requirement I can think of is redesigning the UI and make it mobile friendly.

**Comment by scordio** [comment]
In concrete, one of the following is required from your side:
* Add once the `hacktoberfest` topic to the repository to opt-in to Hacktoberfest --> each PR created in October that is accepted and/or merged will count towards the Hacktoberfest event
* Add an `hacktoberfest-accepted` label to each PR created in October that you consider valuable (GitHub approval and/or merge not mandatory) --> The PR will count towards the event

If either is set, Hacktoberfest will detect and link the contribution automatically, without any further action on your part.

I personally prefer setting the repository topic (e.g., see [assertj/assertj](https://github.com/assertj/assertj)) which I consider less invasive than the label.

**Comment by marchof** [comment]
As I said, I cannot handle the follow up activities (reviewing PRs and maintaining even bigger scope).

But I'm happy to welcome you as a maintainer here 😉

**Comment by scordio** [comment]
Also fine, happy to help with it @marchof 🙂

**Comment by scordio** [comment]
It seems collaborator access doesn't give the privilege to [add a repository topic](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics#adding-topics-to-your-repository). Maybe I can still bother you with adding the `hacktoberfest` topic? 😅

**Comment by marchof** [comment]
Sorry, done.

**Pull request merged**